### PR TITLE
Enhance seeder with additional cipher types and architectural refactorings

### PR DESCRIPTION
**Pull request by theMickster**
💡 **Before starting your review, remember that Rome wasn't built in a day nor will the Seeder be in a single pull request** 

## 🎟️ Tracking

- https://bitwarden.atlassian.net/browse/PM-31013
- https://bitwarden.atlassian.net/browse/PM-31019
- https://bitwarden.atlassian.net/browse/PM-31016
- https://bitwarden.atlassian.net/browse/PM-31403

## 📔 Objective

**Implement required refactoring to decouple and standardize many of the components that make up the Seeder library.** 

- Decomposed monolithic `CipherSeeder.cs`
    - Created type-specific factories
    - Extracted shared encryption logic
    - Created extensions for cipher-type data transformations
- Added support for additional Cipher Types
- Implemented a Distribution System for percentage based selection of mock org data to replace the ad-hoc distribution logic
- Refactored data generators
    - Added GeneratorContext as lazy-loading container for all generators
    - Implemented specific generators as needed. 
    - Ensured all only had a single purpose and meaning
    - Added region-aware data generation (Card, Identity, Username) supporting 6+ geographic regions
- Significantly improved the UserSeeder by removing all responsibility for handling email manging
    -  MangleId is about test orchestration, not user creation.
    - "Create a user with this email" → Factory concern
    - "Prefix emails so test runs don't collide" → Scene/orchestration concern
    - If a Scene needs test isolation, it should own that. If a Recipe doesn't need isolation (like OrganizationWithVaultRecipe using static methods), it shouldn't be forced to.

Code Quality
    - Refactored all factories to consistent `Create()` naming pattern
    - Unified seeder visibility to `internal` with proper `InternalsVisibleTo` for tests
    - Split relationship entities into dedicated seeders (CollectionUserSeeder, GroupUserSeeder)
    - Extracted organizational structure data to `Data/Static/`

Documentation
    - Updated CLAUDE.md to focus on quick reference and pattern decisions
    - Enhanced README.md with Helpers section and folder structure guide
    - Created comprehensive Data/README.md with generator/distribution tables
    - Added DbSeederUtility/CLAUDE.md for CLI-specific guidance

📝 I executed 7-10 local code reviews while working through this pull request using the out-of-the-box Claude Code Review command & our Bitwarden Claude Code command. I resolved various findings and architectural suggestions as I went.

### CODE REVIEW

File: util/Seeder/Data/Distributions/Distribution.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        var cumulative = 0;
+        foreach (var (value, percentage) in _buckets)
+        {
+            cumulative += (int)(total * percentage);
+            if (index < cumulative)
+            {
+                return value;
+            }
+        }
+        return _buckets[^1].Value;
```
Comment (theMickster) [inline util/Seeder/Data/Distributions/Distribution.cs:45]: 📝 Return last bucket for out-of-range indices (graceful degradation). This handles rounding errors where cumulative counts don't exactly match total.

File: test/SeederApi.IntegrationTest/GeneratorContextTests.cs [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        }
+    }
+
+    /// <summary>
+    /// Limited to 5 iterations to avoid a Bogus.Password() infinite loop bug
+    /// that occurs with certain seed/index combinations in WiFi/Database note categories.
+    /// The workaround is a known test workaround that doesn't affect production code.
+    /// </summary>
+    [Fact]
+    public void SecureNote_SameSeed_ProducesSameOutput()
```
Comment (theMickster) [inline test/SeederApi.IntegrationTest/GeneratorContextTests.cs:139]: 📝 Bogus 35.0+ has an edge case where Password() can infinite loop with certain seed/index combinations in WiFi/Database note categories. Limiting iterations is a test-only workaround that doesn't affect production usage where variety is high.

File: util/Seeder/Helpers/StableHash.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+/// </summary>
+internal static class StableHash
+{
+    /// <summary>
+    /// Converts a string to a stable 32-bit integer using SHA256.
+    /// The same input always produces the same output, regardless of process or runtime.
+    /// </summary>
+    /// <param name="value">The string to hash.</param>
+    /// <returns>A deterministic 32-bit integer derived from the input by taking the first 4 bytes as a stable int32.</returns>
+    internal static int ToInt32(string value)
```
Comment (theMickster) [inline util/Seeder/Helpers/StableHash.cs]: 📝 We use SHA256 instead of string.GetHashCode() because GetHashCode() is randomized per-process in .NET Core for security. SHA256 ensures the same domain always produces the same seed across machines/runs for reproducible tests.

File: util/Seeder/Recipes/OrganizationWithVaultRecipe.cs
Comment (theMickster) [inline util/Seeder/Recipes/OrganizationWithVaultRecipe.cs:1]: ♻️ **Upcoming work**

1️⃣ The biggest TODO is to decide how we are going to break down recipes into even more reusable parts. Something like an 'Ingredient' or a 'Step' would be in order. Then, the recipe only calls for 'Ingredients' in an specific order and perhaps passing some state via a context? 

2️⃣ We need to breakdown vault creation into batches; just too much going on.

3️⃣ We need to create a callback feature to pass progress back to the CLI. Waiting with a flashing cursor is awful.

TODO: Implement cipher-to-folder mapping (user personal vaults). Currently all org ciphers exist outside folders. Real users organize personal copies into folders. Need Distribution<int> for "# of ciphers per folder" and logic to assign subsets of org ciphers to user folders.

TODO: Implement user-owned ciphers (non-org)Real users have personal ciphers outside the organization vault. Need option for personal cipher count and generator for non-work-related content (e.g., home WiFi, personal banking, streaming services).

TODO: Implement industry-aware cipher → collection mappings Current: Round-robin assignment (cipher i % collections.length) Future: Semantic grouping (travel sites → Travel collection, banking → Finance collection, devtools → Engineering collection)
Will require industry taxonomy + company categorization enhancement.

TODO: Implement owner/admin distribution
Current: Single owner, all other users are regular members
Realistic: 2-5% owners, 10-15% admins, rest users
Add OwnerCountDistribution.Realistic for multi-owner scenarios and test owner permission edge cases (e.g., last owner cannot leave org).

TODO: Implement organizational policy seeding
Real orgs have policies: MasterPassword, TwoFactorAuthentication, 
SingleOrg, RequireSso, PersonalOwnership, etc.
Add PolicySeeder factory + options for policy distribution to test policy enforcement scenarios in integration tests.

File: util/Seeder/Factories/UserSeeder.cs
Comment (theMickster) [inline util/Seeder/Factories/UserSeeder.cs:1]: ♻️ **Upcoming work**

TODO: Add password override parameter
Currently all seeded users share the same password for dev convenience.
Future: Support per-user or pattern-based passwords (e.g., "Pass-{username}") for more realistic testing scenarios.

File: util/Seeder/Models/CipherViewDto.cs [hunk 4]
```diff
@@ -32,16 +32,16 @@ public class CipherViewDto
     public LoginViewDto? Login { get; set; }
```
Comment (theMickster) [inline util/Seeder/Models/CipherViewDto.cs:32]: ♻️ Potential upcoming work 

TODO: Add TOTP (2FA) support to LoginViewDto
Real logins often have Time-based One-Time Password seeds.
Need TotpSeed property (string, optional) that generators can populate with realistic authenticator seeds (e.g., "otpauth://totp/...")
This enables testing of TOTP generation/validation in vault scenarios.
No clue how to do this yet, but together we can figure it out.

File: util/Seeder/Data/GeneratorContext.cs
Comment (theMickster) [inline util/Seeder/Data/GeneratorContext.cs:1]: Before this class, recipes had 7+ fields for generators and a long InitializeGenerators() method. Now recipes have 1 field (_ctx) andgenerators are accessed via properties.

Design goal: Make adding generators a 3-line change (field, property, done) without touching recipe constructor or initialization logic.

File: util/Seeder/Data/Generators/SshKeyDataGenerator.cs [hunk 5]
```diff
@@ -0,0 +1,142 @@
+﻿using System.Security.Cryptography;
+using Bit.Seeder.Models;
+
+namespace Bit.Seeder.Data.Generators;
+
+/// <summary>
```
Comment (theMickster) [inline util/Seeder/Data/Generators/SshKeyDataGenerator.cs:6]: 📝 SSH keys use a pre-generated pool because RSA.Create() uses system RNG (not our seed). Keys are marked with -----BEGIN FAKE RSA PRIVATE KEY----- headers to prevent accidental production use. Cross-run reproducibility is not guaranteed for SSH key content.

We really just want nice looking fake data; we don't want some testing tool using them. 
That is so far out of scope right now; it's not even on the horizon

File: util/Seeder/Factories/OrganizationSeeder.cs
Comment (theMickster) [inline util/Seeder/Factories/OrganizationSeeder.cs:1]: ♻️**Upcoming work**
No organizational policies are currently configured. We'll add policy seeding (2FA requirements, Master Password policies, etc.).

File: util/Seeder/Data/Distributions/Distribution.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    /// <exception cref="ArgumentException">Thrown when percentages don't sum to 1.0.</exception>
+    public Distribution(params (T Value, double Percentage)[] buckets)
+    {
+        var total = buckets.Sum(b => b.Percentage);
+        if (Math.Abs(total - 1.0) > 0.001)
+        {
+            throw new ArgumentException($"Percentages must sum to 1.0, got {total}");
+        }
+        _buckets = buckets;
+    }
```
Comment (MGibson1) [inline util/Seeder/Data/Distributions/Distribution.cs:25]: todo: just normalize the buckets when you get them to remove the restriction

File: util/Seeder/Data/Distributions/Distribution.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    {
+        var remaining = total;
+        for (var i = 0; i < _buckets.Length - 1; i++)
+        {
+            var count = (int)(total * _buckets[i].Percentage);
+            yield return (_buckets[i].Value, count);
+            remaining -= count;
+        }
+        yield return (_buckets[^1].Value, remaining);
+    }
```
Comment (MGibson1) [inline util/Seeder/Data/Distributions/Distribution.cs:64]: **nit:** I don't think the last bucket is a special case

```suggestion
    /// <summary>
    /// Returns all values with their calculated counts for a given total.
    /// The last bucket receives any remainder from rounding.
    /// </summary>
    /// <param name="total">Total number of items to distribute.</param>
    /// <returns>Sequence of value-count pairs.</returns>
    public IEnumerable<(T Value, int Count)> GetCounts(int total)
    {
        for (var i = 0; i < _buckets.Length; i++)
        {
            var count = (int)(total * _buckets[i].Percentage);
            yield return (_buckets[i].Value, count);
        }
    }
```

File: util/Seeder/Data/Distributions/PasswordDistributions.cs [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+namespace Bit.Seeder.Data.Distributions;
+
+/// <summary>
+/// Pre-configured password strength distributions for seeding scenarios.
+/// </summary>
+public static class PasswordDistributions
+{
+    /// <summary>
+    /// Realistic distribution based on breach data and security research.
+    /// 25% VeryWeak, 30% Weak, 25% Fair, 15% Strong, 5% VeryStrong
```
Comment (MGibson1) [inline util/Seeder/Data/Distributions/PasswordDistributions.cs:12]: where's this data come from?

File: util/Seeder/Helpers/EmailMangler.cs
Comment (MGibson1) [inline util/Seeder/Helpers/EmailMangler.cs:1]: I don't think this class makes sense. I'm having trouble seeing a situation where an email mangler need to store it's own manglemap.

I seems more likely we want a general mangler, not a slew of specific ones. I don't want to overengineer how we mangle stuff. It behooves us to mangle all strings in the same way, but we're limited by strings that are validated both client and server side. The only example I know right now is emails, but I fully suspect others exist too.

File: util/Seeder/Scenes/SingleUserScene.cs [hunk 9]
```diff
@@ -31,22 +35,28 @@ public class Request
...
     {
-        var user = userSeeder.CreateUser(request.Email, request.EmailVerified, request.Premium);
+        var mangler = new EmailMangler();
+        var mangledEmail = mangler.Mangle(request.Email);
+
+        var user = UserSeeder.Create(
+            mangledEmail,
+            passwordHasher,
+            request.EmailVerified,
+            request.Premium);
```
Comment (MGibson1) [inline util/Seeder/Scenes/SingleUserScene.cs]: My plan was for the seeders to contain the knowledge of how a given object needed to be mangled. The thinking was that all the knowledge about the db requirements should be contained in one place. I think it's roughly the same as the need to hash or encrypt data.

I think you moved it because there is a complexity on use case. If we're seeding for human interaction, you don't want to mangle; if it's for test interactions, you do. I suggest handling that by injecting a ManglerService that can be configure to either mangle or not.

File: util/Seeder/Helpers/EmailMangler.cs [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+/// Usage:
+/// <code>
+/// var mangler = new EmailMangler();
+/// var mangledEmail = mangler.Mangle("user@example.com");  // "a1b2c3d4+user@example.com"
+/// var map = mangler.GetMangleMap();  // { "user@example.com": "a1b2c3d4+user@example.com" }
+/// </code>
+/// </remarks>
+public class EmailMangler
+{
+    private readonly MangleId _mangleId = new();
```
Comment (MGibson1) [inline util/Seeder/Helpers/EmailMangler.cs]: This should probably just be the play id, do we want to ever mangle things that aren't associated with a play?

**Comment by theMickster** [comment]
> I think the discrepency on mangling is really the only thing I'd like to get aligned with.
>....

Works for me. I'm for a chat. 
I see the Seeders and Factories differently and needing to know zero state whatsoever. I also see them as internal to the library and should not be injected into a scene (hence my refactoring). If mangling is required, then I see the tracking of the mangling as a responsibility of the Scene or Recipe. 

Also, what other data points other than the email address must be mangled?

File: util/Seeder/Data/Distributions/PasswordDistributions.cs (hunk 8 above)
Comment (theMickster) [reply util/Seeder/Data/Distributions/PasswordDistributions.cs:12]: Claude. 
I didn't waste time researching nor really mind what the total are in the end as I just wanted something that smelled reasonable to make realistic vault data. 
Eventually, we'll have these will be inputs to the recipe.

File: util/Seeder/Helpers/EmailMangler.cs
Comment (theMickster) [reply util/Seeder/Helpers/EmailMangler.cs:1]: It needed to go somewhere because I didn't want it touching the UserSeeder factory because it didn't make the UserSeeder factory reusable by anything that didn't require mangling _and_ it made UserSeeder responsible for things that I didn't see that it should be responsible for; namely any mangling of information. 

I'm okay with a generic mangler. But again. If you only see that we need email now then why not follow YAGI and just leave it as is and refactor later if we really need other "manglers" 🤷🏼

File: util/Seeder/Data/Distributions/Distribution.cs (hunk 6 above)
Comment (theMickster) [reply util/Seeder/Data/Distributions/Distribution.cs:25]: Will you please explain and clarify what you mean here? I'm not tracking ya

File: util/Seeder/Data/Distributions/Distribution.cs (hunk 7 above)
Comment (theMickster) [reply util/Seeder/Data/Distributions/Distribution.cs:64]: Does it still work as it does now with the same distributions? 
If so then I'm fine with the change 😃

File: util/Seeder/Scenes/SingleUserScene.cs (hunk 9 above)
Comment (theMickster) [reply util/Seeder/Scenes/SingleUserScene.cs]: Not all data does nor should be mangled; much of our data that gets generated should not be manged but needs to make realistic business sense.  I think we are also potentially at a point where more things that need mangling need to be in their own 'thing' and away from data that does not need to be mangled.

File: util/DbSeederUtility/CLAUDE.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
+# CLAUDE.md
```
Comment (withinfocus) [inline util/DbSeederUtility/CLAUDE.md]: ⛏️ Is this the new norm? I thought it would have a relevant title.

File: util/DbSeederUtility/CLAUDE.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
```
Comment (withinfocus) [inline util/DbSeederUtility/CLAUDE.md]: ⛏️ More boilerplate that seems like a waste of space and tokens. Isn't this a magic filename that implies instructions?

File: util/Seeder/Data/Distributions/CipherTypeDistributions.cs [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+namespace Bit.Seeder.Data.Distributions;
+
+/// <summary>
+/// Pre-configured cipher type distributions for seeding scenarios.
+/// </summary>
+public static class CipherTypeDistributions
+{
+    /// <summary>
+    /// Realistic enterprise mix based on typical usage patterns.
+    /// 60% Login, 15% SecureNote, 12% Card, 10% Identity, 3% SshKey
```
Comment (withinfocus) [inline util/Seeder/Data/Distributions/CipherTypeDistributions.cs:12]: ℹ️ Seems ... inaccurate to me. Something to ask about internally.

File: util/Seeder/Data/Distributions/PasswordDistributions.cs (hunk 8 above)
Comment (withinfocus) [reply util/Seeder/Data/Distributions/PasswordDistributions.cs:12]: A ⛏️ like my other comment -- and this can just be a future PR to tweak -- but I think / hope this is skewed more toward fair. This makes me want to suggest something more aligned with your "developer" option above and to have additional distributions for "power users" or similar.

File: util/Seeder/Data/Generators/SshKeyDataGenerator.cs (hunk 5 above)
Comment (withinfocus) [reply util/Seeder/Data/Generators/SshKeyDataGenerator.cs:6]: Does the "fake" addition really stop anyone from using it though? And what do scanners say? I would want it to pass and validators our user experience might have first.

File: util/Seeder/Models/CipherViewDto.cs (hunk 4 above)
Comment (withinfocus) [reply util/Seeder/Models/CipherViewDto.cs:32]: There are a good number of ways to use a seed to make TOTPs. This feels especially low-priority to need to test to me, as we're not testing the user's security itself. I suppose some use cases will "need" it but not totally sure what those are, in a hard-required sense.

File: util/Seeder/Recipes/OrganizationWithVaultRecipe.cs
Comment (withinfocus) [reply util/Seeder/Recipes/OrganizationWithVaultRecipe.cs:1]: This deserves itemization in Jira tasks so it's not lost, with discussion / contemplation there.

File: util/Seeder/CLAUDE.md [hunk 14]
```diff
@@ -1,215 +1,82 @@
-# Seeder - Claude Code Context
+# CLAUDE.md
```
Comment (withinfocus) [inline util/Seeder/CLAUDE.md]: ⛏️ Similar comments as the other file.

File: util/Seeder/Recipes/OrganizationWithVaultRecipe.cs
Comment (theMickster) [reply util/Seeder/Recipes/OrganizationWithVaultRecipe.cs:1]: Yes, sir. I have documented them (and others) in https://bitwarden.atlassian.net/browse/PM-31403

File: util/Seeder/Models/CipherViewDto.cs (hunk 4 above)
Comment (theMickster) [reply util/Seeder/Models/CipherViewDto.cs:32]: Sure thing; I wasn't sure where it fit in the priority list. Thanks for the info 😃

File: util/Seeder/Data/Generators/SshKeyDataGenerator.cs (hunk 5 above)
Comment (theMickster) [reply util/Seeder/Data/Generators/SshKeyDataGenerator.cs:6]: No, I don't think it would stop anyone from using them. I will look closer into our static code analyzers to see if anything barks. In terms of the UI, it 'seems' to pass validation on the frontend (as in I don't see any toaster messages) when I open the modal.

File: util/Seeder/Data/Distributions/PasswordDistributions.cs (hunk 8 above)
Comment (theMickster) [reply util/Seeder/Data/Distributions/PasswordDistributions.cs:12]: I agree that with these distributions we can have all sorts of different distributions for all different types of user vaults and organization vaults.  And yes, this is not something to spend too much thought time on at this point. 

We can have the 'boomer' distribution that leans towards the less secure side and then the 'bitwarden' distribution that's on the way secure side.  And these distributions can all be easily added and maintained all outside the main recipes & steps.  😃

File: util/DbSeederUtility/CLAUDE.md (hunk 11 above)
Comment (theMickster) [reply util/DbSeederUtility/CLAUDE.md]: Nope, I just didn't focus on it; I was more focused on the code and kinda let Claude Code have it with the `Claude.md` so long as it didn't go nuts.

File: util/DbSeederUtility/CLAUDE.md (hunk 12 above)
Comment (theMickster) [reply util/DbSeederUtility/CLAUDE.md]: Agreed, it should be cleaned up. Again, I was more focused on the code and kinda let Claude Code have it with the `Claude.md` so long as it didn't go nuts.

File: util/Seeder/Data/Distributions/CipherTypeDistributions.cs (hunk 13 above)
Comment (theMickster) [reply util/Seeder/Data/Distributions/CipherTypeDistributions.cs:12]: Fair point and likely correct. 
Without any access to environments outside my own, I took a random guess to get some data that 'looked' normal with plenty of examples of things other than logins.

File: util/DbSeederUtility/CLAUDE.md (hunk 12 above)
Comment (theMickster) [reply util/DbSeederUtility/CLAUDE.md]: I have decided to remote it all together and, instead, leverage the `Claude.md` in the seeder library because Claude Code should be pulling in the `Claude.md` from the seeder library anyways when anything comes up in this project.

File: util/Seeder/Factories/UserSeeder.cs [hunk 15]
```diff
@@ -13,16 +14,20 @@ internal static class UserSeeder
...
+        IManglerService manglerService,
         bool emailVerified = true,
         bool premium = false,
         UserKeys? keys = null)
     {
-        keys ??= RustSdkService.GenerateUserKeys(email, DefaultPassword);
+        // When keys are provided, caller owns email/key consistency - don't mangle
+        var mangledEmail = keys == null ? manglerService.Mangle(email) : email;
+
+        keys ??= RustSdkService.GenerateUserKeys(mangledEmail, DefaultPassword);
```
Comment (MGibson1) [inline util/Seeder/Factories/UserSeeder.cs:25]: Non-blocking question: Under what circumstances are you expecting user keys to be provided?

Who is meant to produce those keys and how to _they_ mangle to ensure tests can be executed in parallel?

File: util/SeederApi/Extensions/ServiceCollectionExtensions.cs [hunk 16]
```diff
@@ -73,4 +75,23 @@ public static IServiceCollection AddQueries(this IServiceCollection services)
...
+        GlobalSettings globalSettings)
+    {
+        if (globalSettings.TestPlayIdTrackingEnabled)
+        {
+            services.TryAddScoped<IManglerService, ManglerService>();
+        }
+        else
+        {
+            services.TryAddSingleton<IManglerService, NoOpManglerService>();
+        }
```
Comment (MGibson1) [inline util/SeederApi/Extensions/ServiceCollectionExtensions.cs:93]: nice! I like how this injection turned out

File: util/Seeder/MangleId.cs [hunk 17]
```diff
@@ -9,13 +9,7 @@ namespace Bit.Seeder;
 /// </summary>
 public class MangleId
```
Comment (MGibson1) [inline util/Seeder/MangleId.cs]: I'd like to see this as a Private class to the MangleService. We don't want to create a bunch of MangleIds per scene, we want exactly one per scene for consistency with the metaphor: We can have the same actor in different scenes (SeederAPI request), but can have only one instance of an actor within the same scene.

The way this is written begs to be instantiated across many scenes rather than injected in seeder constructors through the MangleService.

Comment (theMickster) [reply util/Seeder/MangleId.cs]: Agreed

File: test/Api.IntegrationTest/AdminConsole/Controllers/OrganizationUsersControllerPerformanceTests.cs [hunk 18]
```diff
@@ -33,7 +34,8 @@ public async Task GetAllUsers_WithCollections(int seats)
         var db = factory.GetDatabaseContext();
         var mapper = factory.GetService<IMapper>();
         var passwordHasher = factory.GetService<IPasswordHasher<User>>();
-        var orgSeeder = new OrganizationWithUsersRecipe(db, mapper, passwordHasher);
+        var manglerService = new NoOpManglerService();
```
Comment (withinfocus) [inline test/Api.IntegrationTest/AdminConsole/Controllers/OrganizationUsersControllerPerformanceTests.cs]: ❓ Why not resolve these with DI? Bring the same setting over to control it.

File: util/SeederApi/Extensions/ServiceCollectionExtensions.cs (hunk 16 above)
Comment (theMickster) [reply util/SeederApi/Extensions/ServiceCollectionExtensions.cs:93]: Me too; this feels more natural.

File: util/Seeder/Factories/UserSeeder.cs (hunk 15 above)
Comment (theMickster) [reply util/Seeder/Factories/UserSeeder.cs:25]: We needed the symmetric key in the Recipe to make folders (at least I'm fairly certain we do). 
And, it just felt a little odd for the Seeder to return a UserWithKeys or some tuple.

File: test/Api.IntegrationTest/AdminConsole/Controllers/OrganizationUsersControllerPerformanceTests.cs (hunk 18 above)
Comment (theMickster) [reply test/Api.IntegrationTest/AdminConsole/Controllers/OrganizationUsersControllerPerformanceTests.cs]: Fixed

**Review by MGibson1 — APPROVED** [review]
